app/models/cart.rb

```diff
@@ -3,5 +3,5 @@ class Cart < ApplicationRecord
   has_many :cart_items, dependent: :destroy
 
   validates :user_id, presence: true
-  validates :item_count, presence: true, numericality: { greater_than_or_equal_to: 0 }
+  validates :item_count, allow_blank: true, numericality: { greater_than_or_equal_to: 0 }
```

Comment:
why did we use `allow_blank: true`?

Reply:
Bhai I added that for temporary cart creation. I changed the entire cart creation logic later, where item_count will be set by the controller.